data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSinkConfig.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.kafka.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.util.List;
+
+/**
+ * * A helper class that helps to read user configuration values from
+ * pipelines.yaml
+ */
+
+public class KafkaSinkConfig {
+
+    @JsonProperty("bootstrap_servers")
+    @NotNull
+    @Size(min = 1, message = "Bootstrap servers can't be empty")
+    private List<String> bootStrapServers;
+
+    @JsonProperty("compression_type")
+    private String compressionType;
+
+    @JsonProperty("batch_size")
+    private Long batchSize;
+
+    @JsonProperty("max_request_size")
+    private String maxRequestSize;
+
+    @JsonProperty("acks")
+    private String acks;
+
+    @JsonProperty("topics")
+    private List<TopicConfig> topics;
```

Comment:
should this be a single topic? when will customer configure multiple topics as sink?

Reply:
As per the requirement, this should be multiple topics.